tests/ignite/engine/test_create_supervised.py

```diff
@@ -125,6 +125,113 @@ def _():
                 trainer.run(data)
 
 
+def _default_create_supervised_trainer_with_output_function(
```

Comment:
Instead of copy-pasting the code I think you can add a new arg to `_default_create_supervised_trainer` and `_test_create_supervised_trainer`. Then you can add a new line with the new arg activated into `test_create_supervised_trainer` to be sure to run the test

Reply:
ok sure I will do that. Should I make changes in the evaluator as well